api/src/main/scala/dcos/metronome/api/v1/controllers/JobScheduleController.scala

```diff
@@ -18,7 +19,8 @@ class JobScheduleController(
   jobSpecService:    JobSpecService,
   val authenticator: Authenticator,
   val authorizer:    Authorizer,
-  val config:        ApiConfig) extends Authorization {
+  val config:        ApiConfig,
+  val mat:           Materializer) extends Authorization {
```

Comment:
I wonder where is this materializer used :)

Reply:
it's there because of the Authorization trait